app/scripts/minmax/dimMinMax.controller.js

```diff
@@ -19,8 +19,16 @@
 
     function getBestArmor(bucket, locked) {
       var armor = {};
+      if (vm.active == 'all'){
+            return armor;
+      }
+
       var best = [], best_non_exotic = [];
       for(var armortype in bucket) {
+          if(armortype === 'all')
```

Comment:
shouldn't need this check since `getBestArmor` won't be called for "all" (per line 360)